Exiled.Events/Patches/Events/Player/SpawningRagdoll.cs

```diff
@@ -43,6 +43,11 @@ private static bool Prefix(RagdollManager __instance, ref Vector3 pos, ref Quate
                 ownerNick = ev.PlayerNickname;
                 playerId = ev.PlayerId;
 
+                Role role = __instance.hub.characterClassManager.Classes.SafeGet(classId);
+                GameObject gameObject = UnityEngine.Object.Instantiate(role.model_ragdoll, pos + role.ragdoll_offset.position, Quaternion.Euler(rot.eulerAngles + role.ragdoll_offset.rotation));
+
+                Exiled.API.Features.Map.RagdollsValue.Add(new API.Features.Ragdoll(gameObject.GetComponent<Ragdoll>()));
```

Comment:
You create a new ragdoll, but never gave it the ragdoll component, therefore GetComponent<Ragdoll>() will return null.

Reply:
I'm gonna rework this event prob.